admiral/pkg/controller/common/common.go

```diff
@@ -727,3 +727,23 @@ func GenerateUniqueNameForVS(originNamespace string, vsName string) string {
 	return newVSName
 
 }
+
+func IsAGateway(item string) bool {
+	gwAssetAliases := GetGatewayAssetAliases()
+	for _, gw := range gwAssetAliases {
+		if strings.HasSuffix(strings.ToLower(item), Sep+strings.ToLower(gw)) {
+			return true
+		}
+	}
+	return false
+}
+
+func GetPartitionAndOriginalIdentifierFromPartitionedIdentifier(partitionedIdentifier string) (string, string) {
```

Comment:
Can you add a comment with an example on how the identifier will look and what is returned thanks

Reply:
+1